yb-voyager/cmd/importData.go

```diff
@@ -1206,6 +1223,33 @@ func extractCopyStmtForTable(table string, fileToSearchIn string) {
 	}
 }
 
+// get the parent table if its a partitioned table in yugabytedb
+// also covers the case when the partitions are further subpartitioned to multiple levels
+func getParentTable(table string, conn *pgx.Conn) string {
+	var parentTable string
+	query := fmt.Sprintf(`SELECT inhparent::pg_catalog.regclass
+	FROM pg_catalog.pg_class c JOIN pg_catalog.pg_inherits i ON c.oid = inhrelid
```

Comment:
Any reason for the alias `pg_catalog.pg_inherits i`?...I don't see it being used anywhere..

Reply:
Didn't use it with `inhrelid` column. Removing it as not needed.